hr_period/models/hr_fiscal_year.py

```diff
@@ -0,0 +1,242 @@
+# -*- coding:utf-8 -*-
+from datetime import datetime
+strftime = datetime.strptime
+from dateutil.relativedelta import relativedelta
+from itertools import chain
+
+from openerp import models, fields, api,  _
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT
+
+
+INTERVALS = {
+    'annually': (relativedelta(months=12), 1),
+    'semi-annually': (relativedelta(months=6), 2),
+    'quaterly': (relativedelta(months=3), 4),
+    'bi-monthly': (relativedelta(months=2), 6),
+    'monthly': (relativedelta(months=1), 12),
+    'bi-weekly': (relativedelta(weeks=2), 26),
+    'weekly': (relativedelta(weeks=1), 52),
+    'daily': (relativedelta(days=1), 365),
+}
+
+@api.model
+def get_schedules(self):
+    return [
+        ('annually', _('Annually (1)')),
+        ('semi-annually', _('Semi-annually (2)')),
+        ('quarterly', _('Quarterly (4)')),
+        ('bi-monthly', _('Bi-monthly (6)')),
+        ('monthly', _('Monthly (12)')),
+        ('semi-monthly', _('Semi-monthly (24)')),
+        ('bi-weekly', _('Bi-weekly (26)')),
+        ('weekly', _('Weekly (52)')),
+        ('daily', _('Daily (365)')),
+    ]
+
+@api.model
+def get_payment_days(self):
+    expr = _('%s day of the next period')
+    expr_2 = _('%s day of the current period')
+    return [
+        ('1', expr % _('First')),
+        ('2', expr % _('Second')),
+        ('3', expr % _('Third')),
+        ('4', expr % _('Fourth')),
+        ('5', expr % _('Fifth')),
+        ('0', expr_2 % _('Last')),
+    ]
+
+
+class HrFiscalYear(models.Model):
+    _name = 'hr.fiscalyear'
+    _description = 'HR Fiscal Year'
+    
+    @api.model
+    def _default_date_start(self):
+        today = datetime.now()
+        return datetime(today.year, 1, 1).strftime(
+            DEFAULT_SERVER_DATE_FORMAT)
+    
+    @api.model
+    def _default_date_stop(self):
+        today = datetime.now()
+        return datetime(today.year, 12, 31).strftime(
+            DEFAULT_SERVER_DATE_FORMAT)
+    
+    name = fields.Char('Fiscal Year', required=True,readonly=True, 
+                       states={'draft': [('readonly', False)]})
+    company_id = fields.Many2one('res.company', 'Company', required=True,
+                                 readonly=True, 
+                                 states={'draft': [('readonly', False)]},
+                                 default=lambda obj: obj.env.user.company_id)
```

Comment:
can you also not indent you code at the end of the line ? this will be much more readable if you skip one line and use one line per parameter. Then, all arguments of every fields will be aligned.